### PR TITLE
remove some cpp that doesn't make sense

### PR DESCRIPTION
**Pull request by shayne-fletcher**
there was some cpp here & there in the examples that wasn't making a lot of sense. this fixes it.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -150,6 +150,23 @@ strategy:
       resolver: "nightly-2020-01-08"
       stack-yaml: "stack.yaml"
 
+    # +---------+-----------------+------------+
+    # | OS      | ghc-lib flavour | GHC        |
+    # +=========+=================+============+
+    # | linux   | ghc-8.10.7       | ghc-8.8.1 |
+    # | macOS   | ghc-8.10.7       | ghc-8.8.1 |
+    # +---------+-----------------+------------+
+    linux-ghc-8.10.7-8.8.1:
```
Comment (shayne-fletcher) [inline azure-pipelines.yml:159]: at some point we dropped testing 8.10 flavor builds. fix that.

**Comment by shayne-fletcher** [comment]
@akrmn

File: examples/mini-hlint/src/Main.hs [hunk 2]
```diff
@@ -168,14 +159,9 @@ fakeSettings = Settings
...
       }
-#if !defined(GHC_MASTER) && !defined(GHC_921)
-    platformConstants =
-      PlatformConstants{
-          pc_DYNAMIC_BY_DEFAULT=False
-        ,
-          pc_WORD_SIZE=8
-    }
 #endif
+#if !defined(GHC_MASTER) && !defined(GHC_941) && !defined(GHC_921)
```
Comment (shayne-fletcher) [inline examples/mini-hlint/src/Main.hs:163]: incidental bugfix here

**Comment by shayne-fletcher** [comment]
@sofiafaro-da

**Review by sofiafaro-da — APPROVED** [review]
Thanks!

**Comment by shayne-fletcher** [comment]
this is looking good @sofiafaro-da . land please? (by the way iirc, @garyverhaegen-da set this repo up so you can stick an automerge label on it at the time you `/azp run`. might be of use to you going forward! :) )

**Comment by sofiafaro-da** [comment]
> this is looking good @sofiafaro-da . land please? (by the way iirc, @garyverhaegen-da set this repo up so you can stick an automerge label on a diff at the time you review/`/azp run`. might be of use to you going forward! 🙂 )

Will do :) thanks for the reminder!

**Pull request merged**